slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -52,6 +52,42 @@ ::grpc::Status SLAMServiceImpl::GetPosition(ServerContext *context,
     return grpc::Status::OK;
 }
 
+// TODO 01/10/2023: Remove current GetPosition and rename this implementation to
```

Comment:
[nit] No need for this TODO, since it won't be hard to find this when implementing the ticket. If you do prefer to keep the TODO, I would write this as:
TODO RSDK-1066: Remove...

Reply:
Removing